pkg/collector/corechecks/oracle-dbm/processes.go

```diff
@@ -37,7 +37,8 @@ type ProcessesRowDB struct {
 
 func (c *Check) ProcessMemory() error {
 	rows := []ProcessesRowDB{}
-	err := c.db.Select(&rows, PGA_QUERY)
+	//err := c.db.Select(&rows, PGA_QUERY)
```

Comment:
I think the commented out can be deleted now.

Reply:
Done.